src/guide/component-registration.md

```diff
@@ -1,5 +1,7 @@
 # Component Registration
 
+<VideoLesson href="https://vueschool.io/lessons/vue-3-global-vs-local-vue-components?friend=vuejs" title="Free Vue.js Component Registration lesson">Learn how component registration works with a free lesson on Vue School</VideoLesson>
```

Comment:
In **vue-3-global-vs-local-vue-components**, around 3:35, it says:

> Since this component does not have a parent, we will define it in our root instance.

I don't think this is quite right. The root component *is* the parent in this case.